src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -215,6 +215,8 @@ internal enum MessageID
         IDS_Return = MessageBase + 12790,
         IDS_FeatureVarianceSafetyForStaticInterfaceMembers = MessageBase + 12791,
         IDS_FeatureConstantInterpolatedStrings = MessageBase + 12792,
+
+        IDS_FeatureUsingTypeAlias = MessageBase + 12800,
```

Comment:
not sure hte number to pick here.  i could make this 12793 if that's appropriate.

Reply:
This number is fine. It may need to be adjusted when merging to main, but likely there won't be a conflict.